presto-iceberg/src/main/java/com/facebook/presto/iceberg/TableStatisticsMaker.java

```diff
@@ -164,20 +184,14 @@ private TableStatistics makeTableStatistics(IcebergTableHandle tableHandle, Cons
         result.setRowCount(Estimate.of(recordCount));
         result.setTotalSize(Estimate.of(summary.getSize()));
         Map<Integer, ColumnStatistics.Builder> tableStats = getClosestStatisticsFileForSnapshot(tableHandle)
-                .map(TableStatisticsMaker::loadStatisticsFile).orElseGet(Collections::emptyMap);
+                .map(this::loadStatisticsFile).orElseGet(Collections::emptyMap);
         for (IcebergColumnHandle columnHandle : selectedColumns) {
             int fieldId = columnHandle.getId();
             ColumnStatistics.Builder columnBuilder = tableStats.getOrDefault(fieldId, ColumnStatistics.builder());
             Long nullCount = summary.getNullCounts().get(fieldId);
             if (nullCount != null) {
                 columnBuilder.setNullsFraction(Estimate.of(nullCount / recordCount));
             }
-            if (summary.getColumnSizes() != null) {
-                Long columnSize = summary.getColumnSizes().get(fieldId);
-                if (columnSize != null) {
-                    columnBuilder.setDataSize(Estimate.of(columnSize));
-                }
-            }
```

Comment:
Since the column size is useless now in summary, should we as well remove the invocation of `updateColumnSizes(...)` in `getSummaryFromFiles(...)` when getting the summary?